### DISCUSSION TITLE
Is this project alive? Or are there forks that are alive?

### DISCUSSION THREAD

**Discussion by deconstructo** [vradarserver/vrs#89] [General]
Hi All,

Running a slightly buggy version of the 3.0.0 beta here..

just wondering - are there forks that are seeing active development?

**Comment by vradarserver** [comment]
I'm not aware of any forks, but then there's no reason why I would be told about them.

I am working on a .NET Core version of the program. I've already done one as a test a couple of years ago and it worked fine, but it was basically just a port that compiled under .NET Core, it didn't take advantage of any of the new stuff or follow the .NET Core way of doing things. This new tack is not a full rewrite, it will reuse quite a lot of code from the .NET Framework version, but parts of it are getting rewritten for .NET Core.

One significant part that will be rewritten is the user interface. .NET Core, unlike .NET Framework, is cross platform. The intention is for the server UI to be web based so that it works the same way on all platforms. Something along the lines of the web admin plugin now.

So no - not dead, but I'm not doing much on the v2 / v3 branches.

**Reply by Geffers** [reply]
I am using V3 beta remotely on a cloud server.  Lack of Web Admin Plugin awkward but I set up remote system locally (won't work as connection details different), I then copy my local linux    /home/<user>/.local/share/VirtualRadar folder to remote location using same filepath and so far seems to work.

Geoff
